### DISCUSSION TITLE
How to disable rubocop warnings or run them on save file?

### DISCUSSION THREAD

**Discussion by marko-avlijas** [castwide/solargraph#541] [Q&A]
Hi,

Thanks for making solargraph. It's pretty amazing.

I want to change one thing though. I am annoyed by constant suggestions / nagging as I am writing code. It breaks my concentration as I am thinking about the code.

I'd like to disable rubocop linting functionality and leave go to definition and other LSP functionality. Alternatively rubocop linting could work on save not all the time as I type.

 How can I do that?

My setup is NVIM v0.6.1 and I followed this article exactly: https://blog.backtick.consulting/neovims-built-in-lsp-with-ruby-and-rails/

**Comment by castwide** [comment]
Try creating a `.solargraph.yml` file at the root of your project. You can run `solargraph config` to create a file with the default settings.

To disable linting completely:

```
reporters: []
```

More information: https://solargraph.org/guides/configuration
